packages/fontless/examples/rsc-app/src/server.css

```
@@ -0,0 +1,3 @@
+.server-poppins {
+  font-family: "Poppins", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)